src/test/java/SpaceIDConverterTest.java

```diff
@@ -0,0 +1,19 @@
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class SpaceIDConverterTest {
+    @Test
+    public void itConvertsAnIntegerToASpace() {
+        SpaceIDConverter converter = new SpaceIDConverter(3, 3);
+        assertEquals(new Space(0, 0), converter.convert(1));
+        assertEquals(new Space(1, 0), converter.convert(2));
+        assertEquals(new Space(2, 0), converter.convert(3));
+        assertEquals(new Space(0, 1), converter.convert(4));
+        assertEquals(new Space(1, 1), converter.convert(5));
+        assertEquals(new Space(2, 1), converter.convert(6));
+        assertEquals(new Space(0, 2), converter.convert(7));
+        assertEquals(new Space(1, 2), converter.convert(8));
+        assertEquals(new Space(2, 2), converter.convert(9));
+    }
```

Comment:
I am still trying to find a source in writing, but in a talk that Michael Feathers gave, he discussed having a hard outer shell to your code (e.g. validations and exceptions) so that the interior logic could be soft and gooey (only expecting valid inputs). In this case, I am ok with only testing the inputs that would be let through by the UI.